types/fluent-ffmpeg/index.d.ts

```diff
@@ -296,9 +297,12 @@ declare namespace Ffmpeg {
         mergeToFile(target: string | stream.Writable, options?: { end?: boolean }): FfmpegCommand;
         concatenate(target: string | stream.Writable, options?: { end?: boolean }): FfmpegCommand;
         concat(target: string | stream.Writable, options?: { end?: boolean }): FfmpegCommand;
+        clone(): FfmpegCommand;
+        run(): void;
     }
 }
 declare function Ffmpeg(options?: Ffmpeg.FfmpegCommandOptions): Ffmpeg.FfmpegCommand;
 declare function Ffmpeg(input?: string | stream.Readable, options?: Ffmpeg.FfmpegCommandOptions): Ffmpeg.FfmpegCommand;
 
+export as namespace Ffmpeg;
```

Comment:
This should only be here if the package creates an `Ffmpeg` global variable. Can you link to documentation for this?